src/index.tsx

```diff
@@ -1009,17 +1009,11 @@ class App extends React.Component<{}, AppState> {
       parsedElements[0].type // need to implement a better check here...
     ) {
       clearSelection(elements);
-
-      if (x == null) x = 10 - this.state.scrollX;
-      if (y == null) y = 10 - this.state.scrollY;
-      const minX = Math.min(...parsedElements.map(element => element.x));
-      const minY = Math.min(...parsedElements.map(element => element.y));
```

Comment:
You can't remove this. It is useful for pasting multiple elements

Reply:
😬just realized it